tfac/figures/figure5.py

```diff
@@ -24,7 +23,7 @@ def makeFigure():
 
 
 def protein_R2X(ax):
-    # data processing for protein
+    """ data processing for protein """
```

Comment:
This doesn't look like a description of this function. Can it be something better?

Reply:
Yup, I'll get on that